tools/bootstrap-plugin-cache.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -euo pipefail
+
+# Disable spinner even when we have a TTY
+export CI='1'
+
+find ./examples -type f -name '*cdktf.json' |  tr '\n' " " | xargs jq -s "map(.terraformProviders) | add | unique | { \"language\": \"typescript\", \"terraformProviders\": . }" > bootstrap.json
```

Comment:
I don't think there's an easy way to represent this on Windows. If it wasn't for this line, I think we could easily convert it to a JS file and make it platform independent. What do you think @jsteinich ?

Reply:
But yeah, doing this in JS as well shouldn't be an issue

Reply:
I'm certainly not aware of any reasonable one-liner in Windows for doing that.

Doesn't seem like it would be too difficult to convert that to JS, though I'm not certain when we'll run this on Windows.

Out of curiosity, does that last pipe (before file) append that block to the output? I don't recall seeing anything like that before.

Reply:
> Out of curiosity, does that last pipe (before file) append that block to the output? I don't recall seeing anything like that before.

Not sure which part you mean

Reply:
`| { \"language\": \"typescript\", \"terraformProviders\": . }`

Reply:
Oh, that's [jq](https://stedolan.github.io/jq/) syntax, where the a new JSON object is constructed and the `.` represents the piped content - which happens to be a list of provider strings at that point.

Reply:
I think this would certainly benefit from not being an obscure one-liner, but more readable Javascript code. Gonna change that.

Reply:
Ah, I missed that it was still within the quotes.